packages/terra-tabs/src/orion-fusion-theme/Tabs.module.scss

```diff
@@ -0,0 +1,90 @@
+:local {
+  .orion-fusion-theme {
```

Comment:
it seems like there are a lot more lowlight theme variables than fusion theme variables, could you double check that  the fusion theme has all back filled variables too? It also helps (me check this) if the files are formatted the same with variables in the same order.

Reply:
Did not add fusion theme styles for Modular tabs because of this [comment](https://github.com/cerner/terra-framework/blob/c66d1bcc63ef184bf4fd589f0dca9c3b83d240d6/packages/terra-tabs/src/Tabs.jsx#L15).Whether the variant will stay or be removed , also there was no example to verify.

Reply:
If thats the case, why did we add the lowlight variables for modular tabs? I don't think they are use since you cannot set tabs to be a modular variant.

Reply:
Had a talk with neil offline and commented out the modular variant styles in both fusion and lowlight after grouping them